examples/tutorials/descriptors_filter.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c13c1dd7",
+   "metadata": {},
+   "source": [
+    "# Running multiple calculations on a given model\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca81fa1b",
+   "metadata": {},
+   "source": [
+    "## Aim"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35ecbb39",
+   "metadata": {},
+   "source": [
+    "This notebook shows how we can run multiple calculations on a given structure"
```

Comment:
These two descriptions are a bit different, and neither is particularly clear.

The focus should be on the types of task/output types we're connecting.